src/inmanta/server/listener.py

```diff
@@ -0,0 +1,54 @@
+"""
+    Copyright 2019 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from inmanta.data import model
+
+
+class EnvironmentListener:
```

Comment:
1. Why move it?
2. this is part of the public api, this will break lsm

Reply:
Because of this: https://github.com/inmanta/inmanta-core/pull/8050#discussion_r1745309622

I'm not sure to see a solution if we don't move something, either the `listener` or the `AutostartedAgentManager`